src/pages/SimpleViewer.jsx

```diff
@@ -127,7 +170,10 @@ export default function SimpleViewer(props) {
                 omitHeader
               />
               {user && (
-                <button type="button" onClick={uploadMessage}>Upload</button>
+                <>
+                  <Button onClick={uploadMessage}>Upload</Button>
```

Comment:
Filled is fine. I intentionally haven't put any effort in styling these because I added them for development purposes, and they will most likely move somewhere else once we have a more fleshed-out UX design.